src/main/java/runner/Storage.java

```diff
@@ -89,4 +80,18 @@ public void saveList() {
             System.out.println(e.getMessage());
         }
     }
+
+    /**
+     * Check the status of the Task and add it to the TaskList.
+     * @param line Description of a Task.
+     * @param t Task checked.
+     */
+    public void check_add(String line, Task t) {
```

Comment:
The variable names can be more clear and descriptive. For example, "line" and "t" could be changed to "taskDescription" and "task", respectively